ansible/roles/screenly/tasks/main.yml

```diff
@@ -70,6 +70,14 @@
     job: "/usr/local/bin/screenly_utils.sh cleanup"
     user: pi
 
+- name: Copy upgrade_screenly.sh
```

Comment:
This is an attack vector. A user could simply add any arbitrary comand into this file and now they can execute anything as root.

Reply:
Updated